pages/common/mkdir.md

```diff
@@ -9,3 +9,7 @@
 - Create directories recursively (useful for creating nested dirs):
 
 `mkdir -p {{path}}`
+
+- Create multiple directories recursively (note the curly braces, marking directory set):
+
+`mkdir -p dir1/dir2/{dir3/{dir4,dir5},dir6,dir7}`
```

Comment:
@igorshubovych the option is the same; but the argument and the effect is different.

@waldyrious why you treat this as out of scope? I suppose this is quite a good alternative to _create multiple directories with one command_